modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Numeric/Numeric.es.js

```diff
@@ -183,4 +183,41 @@ describe('Field Numeric', () => {
 
 		expect(component.dataType).toBe('double');
 	});
+
+	it('check if event is sent when decimal is being writen', done => {
+		const handleFieldEdited = data => {
+			expect(data).toEqual(
+				expect.objectContaining({
+					fieldInstance: component,
+					originalEvent: expect.any(Object),
+					value: expect.any(String)
```

Comment:
@alinedoleron I think it would be better to explicitly say the value we expect here.